LondonBritishLibrary/add/BLadd19337.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:id="BLadd19337" xml:lang="en">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Vocabulary of English, Tigre, and Amharic</title>
+                <editor key="DE"/>
+                <editor role="generalEditor" key="AB"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <pubPlace>Hamburg</pubPlace>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                        <p> This file is licensed under the Creative Commons Attribution-ShareAlike
+                            4.0. </p>
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Additional</collection>
+                        <idno>BL Additional 19337</idno>
+                        <altIdentifier>
+                            <idno>Wright cat. CCCCV</idno>
+                        </altIdentifier>
+                        <altIdentifier>
+                            <idno>Wright 405</idno>
+                        </altIdentifier>
+                    </msIdentifier>
+
+                    <msContents>
+                        <summary/>
+
+                        <msItem xml:id="ms_i1">
+                            <locus from="2r"/>
+                            <title type="complete" xml:lang="en">A vocabulary of English,
+                              <sic>Tigri</sic>, and <sic>Ammerrer</sic>
+                              </title>
+                            <textLang mainLang="en" otherLangs="am ti"/>
```

Comment:
ti is Tigrinya, right? is it here Tigrinya or Tigre?

Reply:
"ti"  is Tigrigna.